cropduster/views/__init__.py

```diff
@@ -466,9 +466,20 @@ def crop(request):
             thumb_data['id'] = thumb_data['id'].pk
 
     preview_url = db_image.get_image_url('_preview')
+    preview_w = PREVIEW_WIDTH
+    preview_h = PREVIEW_HEIGHT
+    orig_width, orig_height = db_image.get_image_size()
```

Comment:
I think what you want here is `db_image.width, db_image.height` (likewise in forms.py for `related_object`)

Reply:
Correction: you want `related_object.width` and `related_object.height` in forms.py, but here you want `crop_data['orig_w']` and `crop_data['orig_h']`